src/tests/test_utils/common_test_utils/include/common_test_utils/matcher.hpp

```diff
@@ -3,6 +3,8 @@
 //
 #pragma once
 
+#pragma once
```

Comment:
the same. duplication of `#pragma once`

Reply:
Probably this is due to rebase side effects. After last rebase second `#pragma once` gone